src/modules/backtest.js

```diff
@@ -309,15 +330,16 @@ module.exports = class Backtest {
       // -- End of Sharpe Ratio Calculation
 
       // Net Profit
-      const netProfit = Number((((workingCapital - initialCapital) / initialCapital) * 100).toFixed(2));
+      const netProfit = Number((((workingCapital - initialCapitalNumber) / initialCapitalNumber) * 100).toFixed(2));
 
       trades.profitabilityPercent = Number(((trades.profitableCount * 100) / trades.total).toFixed(2));
 
       const summary = {
         sharpeRatio: sharpeRatio,
         averagePNLPercent: averagePNLPercent,
         netProfit: netProfit,
-        initialCapital: initialCapital,
+        netFees: cumulativeNetFees,
```

Comment:
netFees would imply there to also to be grossFees

Reply:
I'm sorry, I'm not familiar with the financial term, am I missing more information to be shown in the screen? Thanks!